src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,118 @@
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+import messages from './messages';
+
+import { BackButton } from '../../common/nav-button';
+import Button from '../../../components/button';
+import BoxDrive140 from '../../../illustration/BoxDrive140';
+
+import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
+import StaticVersionsItem from './StaticVersionsItem';
+
+import './StaticVersionsSidebar.scss';
+import './VersionsSidebar.scss';
+import './VersionsItem.scss';
+import './VersionsItemButton.scss';
+import './VersionsGroup.scss';
+import '../SidebarContent.scss';
+
+type Props = {
+    isLoading: boolean,
+    onUpgradeClick?: () => void,
+    parentName: string,
+    showUpsellWithPicture: boolean,
+};
+
+const StaticVersionsSidebar = ({ onUpgradeClick, isLoading, parentName, showUpsellWithPicture }: Props) => {
+    const elementId = '';
+    const sidebarView = '';
+    const label = `${elementId}${elementId === '' ? '' : '_'}${sidebarView}`;
+    const id = `${label}-content`;
+
+    return (
+        <div
+            aria-labelledby={label}
+            className={classNames('bcs-content', 'bcs-Versions')}
+            data-testid="bcs-content"
+            id={id}
+            role="tabpanel"
+            data-resin-component="preview"
+            data-resin-feature="versions"
+        >
+            <div className="bcs-content-header">
+                <h3 className="bcs-title">
+                    <>
+                        <BackButton data-resin-target="back" to={`/${parentName}`} />
+                        <FormattedMessage {...messages.versionsTitle} />
+                    </>
+                </h3>
+            </div>
+            <div className="bcs-scroll-content-wrapper">
+                <div className="bcs-scroll-content">
+                    <LoadingIndicatorWrapper
+                        className="bcs-Versions-content"
+                        crawlerPosition="top"
+                        isLoading={isLoading}
+                    >
+                        <div className="bcs-Versions-menu">
+                            <ul className="bcs-VersionsMenu">
+                                <li className="bcs-VersionsMenu-item" key="versionsToday">
+                                    <section className="bcs-VersionsGroup">
+                                        <h4 className="bcs-VersionsGroup-heading">
+                                            <FormattedMessage {...messages.versionsToday} />
+                                        </h4>
+                                        <ul className="bcs-VersionsList">
+                                            <li className="bcs-VersionsList-item" key={1}>
+                                                <StaticVersionsItem
+                                                    versionNumber="1"
+                                                    versionTime="2021-11-29T15:01:19-08:00"
```

Comment:
seems like the versionTimes are all the same timestamp, should we save this as a constant above and re-use for all three?

Also thoughts on using the new Date() aka today as the fake time string?

Reply:
I like the idea of using Date! That will keep the date more relevant.